logsmanagement/README.md

```diff
@@ -0,0 +1,676 @@
+* When collected logs contain parsable timestamps, these will be used to display metrics from parsed logs at the correct time in each chart, even if collection of said logs takes place *much* later than the time they were produced. How much later? Up to a configurable value of `update timeout` seconds. This mechanism ensures correct  parsing and querying of delayed logs that contain parsable timestamps (such as streamed inputs or buffered logs sources that write logs in batches), but the respective charts may lag behind some seconds up to that timeout. If no parsable timestamp is found, the collection timestamp will be used instead (or the collector can be forced to always use the collection timestamp by setting `use log timestamp = no`).
+
+<a name="collector-types"/>
+
+### Types of available log collectors
+
+</a>
+
+The following log collectors are supported at the moment. The table will be updated as more collectors are added:
+|  Collector    	| Log type      		| Description  |
+| ------------  	| ------------  		| ------------ |
+| kernel logs (kmsg)| `flb_kmsg`    		| Collection of new kernel ring buffer logs.|
+| systemd       	| `flb_systemd` 		| Collection of journald logs.|
+| docker events 	| `flb_docker_events` 	| Collection of docker events logs, similar to executing the `docker events` command.|
+| tail				| `flb_tail` 			| Collection of new logs from files by "tailing" them, similar to `tail -f`.|
+| web log       	| `flb_web_log` 		| Collection of Apache or Nginx access logs.|
+| syslog socket   	| `flb_syslog`  		| Collection of RFC-3164 syslog logs by creating listening sockets.|
+| serial        	| `flb_serial`  		| Collection of logs from a serial interface.|
+| mqtt           	| `flb_mqtt`    		| Collection of MQTT messages over a TCP connection.|
+
+<a name="getting-started"/>
+
+## Getting Started
+
+</a>
+
+Since version `XXXXX`, Netdata is distributed with built-in logs management functionality, but it is disabled by default and must be explicitly enabled at runtime by setting the respective configuration option in `netdata.conf` to `yes` and restarting the agent:
+```
+[logs management]
+	enabled = yes
+```
+
+There are some pre-configured log sources that Netdata will attempt to automatically discover and monitor that can be edited using `./edit-config logsmanagement.conf` in Netdata's configuration directory.
+
+To get familiar with the Logs Management functionality, the user is advised to read at least the [Summary](#summary) and the [General Configuration](#general-configuration) sections and also any [Collector-specific Configuration](#collector-configuration) subsections, according to each of their use cases.
```

Comment:
I am not sure about the `at least` here (ping @Ancairon ), because after `at least` we suggest reading three sections. Probably, we also need to add commas before `and`.